webapp/src/components/router_showcase/router_showcase.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+    Switch,
+    Route,
+    useLocation,
+} from 'react-router-dom';
+import {getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
+import {useSelector} from 'react-redux';
+
+import {id} from 'src/manifest';
+
+export default function RouterShowcase() {
+    const currentTeam = useSelector(getCurrentTeam);
+    const query = useQuery();
+
+    return (
+        <Switch>
+            <Route path={`/${currentTeam.name}/${id}/teamtest/subpath`}>
+                <h3>{'Hello, Subpath Component!'}</h3>
+            </Route>
+            <Route path={`/${currentTeam.name}/${id}/teamtest/subpath-with-query`}>
```

Comment:
Just for the sake of readability, can we refactor the links in the `Route`s into a function so we can make something to `teamUrlPath('/teamtest/subpath')`?